blockbase/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "blockbase",
-  "version": "1.0.1",
+  "version": "1.1.2",
```

Comment:
Why 1.1.2, not 1.1.0?

Reply:
because the packaje.json wasn't up to date, the stylesheet was 1.0.2

Reply:
right but i think we should go back to 0 in the third position when the second position increases...

Reply:
oh yeah, I don't know what I was thinking! Let me change it